pkgs/top-level/emscripten-packages.nix

```diff
@@ -0,0 +1,107 @@
+{ pkgs }:
+
+# emscripten toolchain abstraction for nix
+# https://github.com/NixOS/nixpkgs/pull/16208
+
+with pkgs; rec {
+  json_c = pkgs.json_c.override {
+    stdenv = emscriptenStdenv;
+  };
+  
+  libxml2 = (pkgs.libxml2.override {
+    stdenv = emscriptenStdenv;
+    supportPython = false;
+  }).overrideDerivation
+    (old: { 
+      buildInputs = old.buildInputs ++ [ autoreconfHook pkgconfig zlib nodejs ];
+      nativeBuildInputs = old.nativeBuildInputs ++ [ zlib pkgconfig ];
+      # just override it with nothing so it does not fail
+      autoreconfPhase = "echo autoreconfPhase not used..."; 
```

Comment:
Normally this phase doesn’t exist, maybe it is enabled by having `autoreconfHook` in the dependencies?